packages/block-library/src/media-text/edit.js

```diff
@@ -282,6 +294,7 @@ function MediaTextEdit( {
 					focalPoint: undefined,
 					mediaWidth: 50,
 					mediaSizeSlug: undefined,
+					featuredImageLink: false,
```

Comment:
I wonder if this should be toggled to false if this flow happens:

- set some image
- set featured image
- set link to post type on
- set some other image
- set featured image
- should link to post type be off now?

Reply:
Yeah it should be off I think...

Reply:
But it wont matter I guess for this PR if I disable the link option.